app/models/user.rb

```diff
@@ -238,6 +242,19 @@ def unread_issues
     issues.unread_by(self)
   end
 
+  def validate_steamid
+    errors.add :steamid unless self.steamid.nil? ||
+      (m = self.steamid.match(/\A([01]):([01]):(\d{1,10})\Z/)) &&
```

Comment:
Align the operands of a condition in an `unless` statement spanning multiple lines.<br>Redundant `self` detected.